src/components/course-settings/remove-teacher.cjsx

```diff
@@ -25,10 +24,11 @@ module.exports = React.createClass
       .value()
 
   goToDashboard: ->
-    @context.router.transitionTo('dashboard', {courseId: @props.courseId})
+    TeacherRosterStore.once 'deleted', ->
+      window.location.href = '/dashboard/'
```

Comment:
hmm, does the following not work (combining `transitionTo` without the course id but wrapped in a `.once 'deleted'`)?

```coffee
TeacherRosterStore.once 'deleted', =>
  @context.router.transitionTo('dashboard')
```